### ISSUE TITLE
Timepicker container parameter ignored

### ISSUE THREAD

**Issue by sebtouze** [Dogfalo/materialize#5249]
It seems that the Pickadate container parameter does not work. Like when you do `$('#timepicker').pickatime({container: 'body'});`
The same works perfectly fine with datepicker. 

## Expected Behavior

With parameter container set, attach the timepicker node to the given root. 

## Current Behavior
Container parameter is ignored and the timepocker node is set after the input. 

## Steps to Reproduce 
See bellow example. The datepicker is attached to body but not the timepicker. 
```html
<!doctype html>
<html class="no-js" lang="fr">
<head>
<link rel="stylesheet" href="materialize.min.css">
</head>
<body>
<form>
  <input id="datepicker" name="datepicker" required="required" class="" value="2017/05/04" tabindex="81" type="date">
  <input id="timepicker" name="timepicker" required="required" class="" value="22:00" tabindex="81" type="time">
</form>
<script src="jquery-1.12.0.min.js"></script>
<script src="materialize.min.js"></script>
<script>
  (function($){
    $(function(){
    $('#datepicker').pickadate({
      selectMonths: true,
      selectYears: 15, 
      closeOnSelect: true, 
      format: 'd mmmm, yyyy',
      formatSubmit: 'yyyy-mm-dd',
      hiddenName: true,
      container: 'body'});
    $('#timepicker').pickatime({
      twelvehour: false,
      autoclose: true,
      vibrate: false,
      container: 'body'
    });
  });})(jQuery);
</script>
</body>
</html>
```

## Context

I want to to that to display a timepicker in a modal window, in case of a low resolution screen I want the time picker to use maximum space and not to be constrained inside Modal. I was trying to reproduce solution as in #1032 (which indeed work well for datepicker).

**Comment by tomscholz** [comment]
Duplicate of https://github.com/Dogfalo/materialize/issues/5225

**Comment by Dogfalo** [comment]
Fixed in 1.0 alpha
